x-pack/plugins/index_management/__jest__/client_integration/home/index_templates_tab.helpers.ts

```diff
@@ -6,18 +6,21 @@
 
 import { ReactWrapper } from 'enzyme';
 import { act } from 'react-dom/test-utils';
+
 import {
   registerTestBed,
   TestBed,
   TestBedConfig,
   findTestSubject,
   nextTick,
 } from '../../../../../test_utils';
+// NOTE: We have to use the Home component instead of the TemplateList component because we depend
+// upon react router to provide the name of the template to load in the detail panel.
```

Comment:
Something around those lines should work

```js
const CompToTest = () => {
  return (
    <Switch>
      <Route exact path="templates/:templateName*" component={TemplateList} />
     </Switch>
  );
};

const testBedConfig: TestBedConfig = {
  memoryRouter: {
    initialEntries: ["templates"],
    componentRoutePath: "templates",
  },
  doMountAsync: true,
};

const setup = registerTestBed(WithAppDependencies(CompToTest), testBedConfig);
```

Reply:
Thanks, Seb! I took a stab at this and every test started failing. I'm going to merge this in its current state to enable us to add tests using this new organization to data streams etc, with the expectation that we'll address your comments separately.